packaging/windows/sharedframework/sharedframework.wxs

```diff
@@ -11,6 +11,27 @@
         <Feature Id="MainFeature" Title="Main Feature" Level="1">
             <ComponentGroupRef Id="InstallFiles" />
             <ComponentGroupRef Id="AuthoredRegistryKeys"/>
+            <Component Id="BreadcrumbStoreFolder" Directory="BreadcrumbStore" Guid="{DA957490-EA59-42B9-B9D6-3E80F6FB0ED0}">
```

Comment:
I see, it's trying to enforce uniqueness.  That's why you force folks to have single file components.  I suppose it still isn't foolproof.  You can still have two components with different registry value keypaths but overlapping values since it doesn't enforce the "single registry value" rule.